packages/docs/src/helpers/analyticsPropTable.tsx

```diff
@@ -0,0 +1,36 @@
+export const analyticsPropTable = [
+  {
+    name: 'analytics',
+    type: 'boolean',
+    defaultValue: 'true',
+    text: 'Analytics events tracking is enabled by default. Set this value to `false` to disable tracking for this component instance.',
```

Comment:
Using `description` doesn't work as that is MDX data that gets compiled, can't compile a string.. Couldn't figure out a way to convert a string to MDX easily at runtime, if it's possible at all, so adding another field here.